Makefile

```diff
@@ -14,6 +14,7 @@ build:
 generate:
 	go get github.com/progrium/go-extpoints
 	go get github.com/jonasfj/go-import-subtree
```

Comment:
sure... but we'll need to update references too... can we do it separately...

Note: added you can garndt as collaborators for now if you have any hacks for it...
I'm very much trying to keep `go-import-subtree` super generic and not taskcluster specific, which I why I wasn't exactly sure where to put it...